distribution/src/assembly/assembly-descriptor-slim.xml

```diff
@@ -71,6 +71,11 @@
             <destName>hazelcast.xml</destName>
             <outputDirectory>config</outputDirectory>
         </file>
+        <file>
+            <source>${configs.from.jar}/hazelcast-docker-assembly.xml</source>
+            <destName>hazelcast-docker.xml</destName>
```

Comment:
Are we going to update the docker image with this file as default? Or how is it going to be used?

Reply:
Yes, we'll update the Docker image too. This is the PR sent on the docker image side: hazelcast/hazelcast-docker#301